src/routes/+page.svelte

```diff
@@ -185,6 +189,10 @@
 						{#if movie?.showtimes} <Showtimes showtimes={movie?.showtimes} /> {/if}
 					</div>
 				</div>
+
+				<div
+					class="absolute inset-x-0 bottom-0 h-24 bg-gradient-to-t from-black z-40 pointer-events-none"
+				/>
```

Comment:
Is it possible that this code causes the modal to lose its round shape on the bottom?


https://user-images.githubusercontent.com/4436129/231550695-ce94c9b3-ed01-4671-887c-c830fb6c3b77.MP4

Reply:
![IMG_3461](https://user-images.githubusercontent.com/4436129/231550938-7a45a213-79bd-4706-b913-68029f2a1028.jpg)

The above video doesn't really show this well. See the red circles ⭕

Reply:
![image](https://user-images.githubusercontent.com/701/231555175-ec12215b-a414-419b-9691-223fe110238d.png)

Reply:
Are you seeing it in this build?

Reply:
This is the link I visited on my iPhone 14 Pro https://hvaderibio-b0ee6qdg5-multivac61.vercel.app/

The behavior is different on Firefox on macOS and iOS Safari 🤔

![CleanShot 2023-04-12 at 19 06 04](https://user-images.githubusercontent.com/4436129/231559861-b7f0c646-105a-4f65-aefb-32d1ecd3c1f4.png)

![IMG_E71224B737BE-1](https://user-images.githubusercontent.com/4436129/231560015-71c44d56-49d1-460f-b148-1b77fa64b8ce.jpeg)

Reply:
Its the same on https://hvaderibio-git-fork-jokull-css-tweaks-multivac61.vercel.app/

Reply:
Not seeing it on iOS, Firefox macOS or Safari macOS. Weird.

Reply:
I got some screenshots from Android users. The first two look like yours..

![Screenshot_20230413-092810](https://user-images.githubusercontent.com/4436129/231717029-406012b7-a1c1-41fb-86d7-8ef0a845d54e.png)
![Screenshot_20230413_092322_com android chrome](https://user-images.githubusercontent.com/4436129/231717041-e41b5758-e48f-47cc-a8c9-7b8f52298ef6.jpg)

The third one is... well... 

![Screenshot_20230413-092702_Samsung Internet](https://user-images.githubusercontent.com/4436129/231717106-37909ad8-6c55-46b1-adab-2757517531b6.jpg)